edweiss-firebase/functions/src/functions/memento/createDeck.ts

```diff
@@ -7,11 +7,14 @@ import { ok } from 'utils/status';
 
 export const createDeck = onSanitizedCall(Memento.Functions.createDeck, {
 	deck: Predicate.fields({
+		ownerID: Predicate.forEach(Predicate.isNonEmptyString),
 		name: Predicate.isNonEmptyString,
 		cards: Predicate.forEach(CustomPredicateMemento.isValidCard)
-	})
+	}),
+	courseId: Predicate.isNonEmptyString
 }, async (userId, args) => {
-	const deckCollection = CollectionOf<Memento.Deck>("decks");
+	//const deckCollection = CollectionOf<Memento.Deck>("decks");
```

Comment:
Great use of sanitizations! I just wanted to highlight this commented code. I think you can remove it. That's just fine like this.